### DISCUSSION TITLE
How to validate if wallet connected with nodejs?

### DISCUSSION THREAD

**Discussion by aleksandr-ideasoft** [WalletConnect/walletconnect-monorepo#2350] [Developer Support]
User connect his wallet on frontend side, and on nodejs I want to validate that specific address connected to frontend. How to do it without additional signed message by wallet?

**Comment by finessevanes** [comment]
Your comment has been viewed and shared with our development team for review.

**Reply by finessevanes** [reply]
Just want to follow up - you are using the Auth client?

**Reply by aleksandr-ideasoft** [reply]
I use SignClient on frontend, because need to send transactions. On backend side it doesn't matter, because I only want to validate that user connect wallet and see wallet address. Also for now I need to connect safe global wallets(gnozis), and seems this wallet dont work with auth client, but work with sign client. Based on this SignClient is preferred option.

**Reply by finessevanes** [reply]
Auth is used for authenticating off-chain. What are you trying to build? Are you using wagmi, ethers, viem or are you handling the providers on your own?

**Comment by finessevanes** [comment]
Closing this discussion due to three weeks of inactivity
